### DISCUSSION TITLE
8K video couldn't be displayed in the gallery

### DISCUSSION THREAD

**Discussion by iamydp** [photoprism/photoprism#2924] [Q&A]
I just uploaded the origin video file into the folder and rescaned, then nothing happened. After viewing the log I found that Photoprism couldn't discover this video file at all. I tried other 4k videos and it worked fine. I also used ffmpeg to transcode it from h265 to h264 but the question remained. Here is the video info.
![image](https://user-images.githubusercontent.com/26623485/202888542-f0c8546f-1a5b-4821-8be3-b102e8f9e500.png)

**Comment by lastzero** [comment]
It's probably too large. Did you try to increase the file size limit yet?
